onomy-testnet1 chain docs

sunnyk56 opened the pull request.

sunnyk56 committed:
update onomy-testnet1 chain documentation

sunnyk56 committed:
update onomy-testnet1 chain documentation

dzmitryhil commented on deploy/testnet/bin.sh:
```diff
@@ -31,7 +31,7 @@ echo "----------------cloning_repository-------------------"
 GRAVITY_DIR=$HOME_DIRECTORY/gravity
 ONOMY_DIR=$HOME_DIRECTORY/go/onomy
-git clone https://github.com/onomyprotocol/cosmos-gravity-bridge.git $GRAVITY_DIR
+git clone -b v0.0.0-20210915184851-orch-nomarket https://github.com/onomyprotocol/cosmos-gravity-bridge.git $GRAVITY_DIR
```
I don't think it is a good idea to set that branch here because it might be removed. so we should either use commit or just the latest master.

dzmitryhil replied:
I would stick to the master since we have this note that it might be unsafe.

dzmitryhil replied:
Or with the tag

dzmitryhil commented on deploy/testnet/master-validator/init.sh:
```diff
@@ -143,4 +154,4 @@ echo "delete faucet account if we have previously"
 echo "Starting faucet based on validator account"
-faucet -cli-name=$GRAVITY -keyring-backend=test -mnemonic="$ONOMY_ORCHESTRATOR_MNEMONIC" &
\ No newline at end of file
+faucet -cli-name=$GRAVITY -keyring-backend=test -mnemonic="$ONOMY_ORCHESTRATOR_MNEMONIC" credit-amount=100000000  -max-credit=200000000 &
```
Have you tested the faucet with those params?

sunnyk56 replied:
I am trying to test the faucet using the flag credit-amount=100,000,000 -max-credit=200,000,000
if I try for 200,000,001 goes fail but when trying call request 2,000,000,000 it goes successful.

I have raised the issue with that https://github.com/tendermint/faucet/issues/10

dzmitryhil commented on docs/testnet/onomy-testnet-docs/faucet.md:
```diff
@@ -42,44 +43,74 @@ Note 'myvalidatorkeyname' is just the name of your key here, you can pick anythi
-wget http://147.182.128.38:26657/genesis? -O $HOME/raw.json
-jq .result.genesis $HOME/raw.json >> $HOME/onomy/onomy/config/genesis.json
+rm $HOME/onomy-testnet1/onomy/config/genesis.json
+wget http://147.182.190.16:26657/genesis? -O $HOME/raw.json
```
As I see you nad to update the docs, because of the change of the IP. So maybe we could you DNS?

sunnyk56 replied:
once DNS testing done, accordingly will update ip to DNS url

dylanschultzie commented:
I have some recommended changes here. I think there should be a restructuring of the docs, such that it goes something like:
- `install onomy`
- `setup node`
   |--> `use faucet`
   |--> `create full validator`

I've tested up to using the faucet, but I can't get it working. Everything up to that is confirmed to work. @dzmitryhil @sunny960 

[creating a validator.md](https://github.com/onomyprotocol/onomy/files/7263273/creating.a.validator.md)
[full node.md](https://github.com/onomyprotocol/onomy/files/7263274/full.node.md)
[install onomyd.md](https://github.com/onomyprotocol/onomy/files/7263275/install.onomyd.md)
[request tokens from faucet.md](https://github.com/onomyprotocol/onomy/files/7263276/request.tokens.from.faucet.md)

dylanschultzie commented:
Dang that didn't appear to work. I'm going to pull your branch and add the files. @sunny960

dylanschultzie commented:
@sunnyk56 @dzmitryhil I've created a PR with the changes. https://github.com/onomyprotocol/onomy/pull/17 

The goal here was to simplify the process, especially using daemons/systemd.

sunnyk56 committed:
1.remove dublication
2 instead of providing the home directory path use default ~/.onomy in the setup-fullnode-manual.md and setup-validator-manual.md

dylanschultzie commented on docs/testnet/onomy-testnet-docs/pre-installation.md:
````diff
@@ -0,0 +1,35 @@
+To download binary follow these commands
+
+```
+cd $HOME
````
I'd *highly* encourage you to adopt the method I outlined where you:
1. install go
2. pull repo
3. make install

as that's the standard within the cosmos community. Adding this overhead for getting binaries will add confusion.

sunnyk56 replied:
instead of making their own binary, the binary link is provided to download and move on.
If users want to make their own binary all these steps all already written in the script file.

sunnyk56 committed:
update ip to host (testnet1.onomy.io)

sunnyk56 commented:
> Dang that didn't appear to work. I'm going to pull your branch and add the files. @sunny960

In the binary script file, I have installed `screen`. basically using screen command new terminal window is created as a separate thread for the validator, orchestrator, and Geth node

sunnyk56 commented:
> I have some recommended changes here. I think there should be a restructuring of the docs, such that it goes something like:
> 
> * `install onomy`
> * `setup node`
>   |--> `use faucet`
>   |--> `create full validator`
> 
> I've tested up to using the faucet, but I can't get it working. Everything up to that is confirmed to work. @dzmitryhil @sunny960
> 
> [creating a validator.md](https://github.com/onomyprotocol/onomy/files/7263273/creating.a.validator.md) [full node.md](https://github.com/onomyprotocol/onomy/files/7263274/full.node.md) [install onomyd.md](https://github.com/onomyprotocol/onomy/files/7263275/install.onomyd.md) [request tokens from faucet.md](https://github.com/onomyprotocol/onomy/files/7263276/request.tokens.from.faucet.md)

I have tested all the things(full node, validator node, and faucet), and it's working fine. If you are facing any issues just send us the description of the steps you have followed and the error messages that you are getting.
